bundles/persistence/org.openhab.persistence.jdbc/lib/.gitignore

```diff
@@ -0,0 +1,3 @@
+/commons-dbutils-1.6.jar
```

Comment:
please put wildcards like "commons-dbutils.*" in this gitignore so this file does not need to be updated with each library update